core-java-modules/core-java-20/src/test/java/com/baeldung/convertpaths/RelativePathConverterUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.convertpaths;
+
+import org.junit.Test;
+
+public class RelativePathConverterUnitTest {
+
+    @Test
+    public void testConvertToAbsolutePath() {
```

Comment:
@vunamtien let's follow the guidelines for tests naming and conventions:
https://docs.google.com/document/d/1iIeSPuYFG-jEtB8UbcxQnn2AI3OR2LWlnisNo2NUCL8/edit